.github/workflows/ci.yml

```diff
@@ -54,12 +56,70 @@ jobs:
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v1
 
+  # API Definition testing
+  # https://docsapi.io/cli
```

Comment:
What is this https://docsapi.io/cli _meant_ to be? 
![image](https://user-images.githubusercontent.com/194400/213881280-1167614f-f537-4897-98fe-98316dc61e14.png)

did you mean: https://docs.hoppscotch.io/cli ?
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/194400/213881359-3450d2d7-0369-4556-ac15-17b1bde14602.png">

Reply:
I did, I think I misscopied.

Reply:
Cool. I've updated. 🔗